packages/web-twig/CONTRIBUTING.md

```diff
@@ -65,4 +65,24 @@ Run `composer phpunit:update` or `make phpunit-update` to create or update snaps
 
 ## Release new version
 
-- run script `. ./newVersion.sh <version>`
+TODO: [Automate release process](https://github.com/lmc-eu/spirit-design-system/issues/393).
+
+As we are now using only `dev-main` version of this package, until we have stable release process, we are not able to release new version with release script and tag it.
+
+- Do not run script `. ./newVersion.sh <version>`
```

Comment:
Could you either join it with the previous sentence, or maybe emphasize it if it deserves attention? It's weird to see it as a single-item list… 🤔 

```suggestion
As we are now using only `dev-main` version of this package, until we have stable release process, we are not able to release new version with release script and tag it. **Do not run the script `./newVersion.sh <version>`.**
```

or

```suggestion
As we are now using only `dev-main` version of this package, until we have stable release process, we are not able to release new version with release script and tag it.

⚠️ **DO NOT RUN** the script `./newVersion.sh <version>`!
```

Reply:
Second choice. Thanks.